src/test/TaskListTest.java

```diff
@@ -0,0 +1,54 @@
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class TaskListTest {
+    private static List<Task> testList;
+
+    @BeforeAll
+    static void setUpList() {
+        testList = new ArrayList<>();
+    }
+
+
+    @Test
+    void addTask_newToDo_success() {
+        TaskList taskList = new TaskList(testList);
+        taskList.addTask("todo borrow book");
+        assertEquals(1, taskList.getNewStorage().size());
+    }
+
+    /*
```

Comment:
Will it be better to remove codes commented out when pushed to master? If necessary, can store them in another branch:)